### PR TITLE
Docsp 27849 connection monitoring

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-27849
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-27849-connection-monitoring/fundamentals/monitoring/connection-monitoring/#event-descriptions

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/monitoring/connection-monitoring.txt [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+   * - ``connectionCheckOutFailed``
+     - Created when an operation fails to acquire a connection for
+       execution.
+
+   * - ``connectionCheckedOut``
+     - Created when an operation successfully acquires a connection for
+       execution.
+
+   * - ``connectionCheckedIn``
+     - Created when a connection is returned to the pool after operation
```
Comment (DBirtolo-mdb) [inline source/fundamentals/monitoring/connection-monitoring.txt:80]: [Question]
This is the only one of these events that does not make sense to me off of reading the description. I am not sure if this is something that warrants more explanation? I leave that up to your judgment, but I thought I would call it out as "returning to the pool" isn't something that made sense to me with my current knowledge set.

**Review by DBirtolo-mdb — APPROVED** [review]
LGTM with one question/call out that does not restrict approval.

File: source/fundamentals/monitoring/connection-monitoring.txt (hunk 1 above)
Comment (rustagir) [reply source/fundamentals/monitoring/connection-monitoring.txt:80]: from my understanding, checked in means checked _back_ in-- the connection is made available again from the pool after an operation is executed. does that make sense?

Comment (DBirtolo-mdb) [reply source/fundamentals/monitoring/connection-monitoring.txt:80]: It does. With that context, I think the description is fine.

**Comment by rustagir** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v5.0|[](https://github.com/mongodb/docs-node/pull/532)|
|✅|v4.14|[](https://github.com/mongodb/docs-node/pull/533)|
|✅|v4.13|[](https://github.com/mongodb/docs-node/pull/534)|
|✅|v4.12|[](https://github.com/mongodb/docs-node/pull/535)|
|✅|v4.11|[](https://github.com/mongodb/docs-node/pull/536)|
|❌|v4.10|An unhandled error occurred. Please see the logs for details|
|❌|v4.9|An unhandled error occurred. Please see the logs for details|
|✅|v4.8|[](https://github.com/mongodb/docs-node/pull/537)|
|✅|v4.7|[](https://github.com/mongodb/docs-node/pull/538)|
|✅|v4.6|[](https://github.com/mongodb/docs-node/pull/539)|
|✅|v4.5|[](https://github.com/mongodb/docs-node/pull/540)|
|✅|v4.4|[](https://github.com/mongodb/docs-node/pull/541)|
|❌|v4.3|Could not create pull request: You have exceeded a secondary rate limit and have been temporarily blocked from content creation. Please retry your request again later.|
|❌|v4.2|Could not create pull request: You have exceeded a secondary rate limit and have been temporarily blocked from content creation. Please retry your request again later.|
|❌|v4.1|Could not create pull request: You have exceeded a secondary rate limit and have been temporarily blocked from content creation. Please retry your request again later.|
|❌|v4.0|Could not create pull request: You have exceeded a secondary rate limit and have been temporarily blocked from content creation. Please retry your request again later.|

Note: Successful backport PRs will be merged automatically after passing CI.

### Manual backport
To create the backport manually run:
```
backport --pr 531
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**